src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/type/TelemetryEventType.kt

```diff
@@ -19,4 +19,8 @@ enum class TelemetryEventType(val eventName: String) {
   NOMIS_PARTIAL_MATCH_FOUND("CprNomisPartialMatchFound"),
   NOMIS_NO_MATCH_FOUND("CprNomisNoMatchFound"),
   NOMIS_PNC_MISMATCH("CprNomisPncMismatchFound"),
+  NEW_DELIUS_RECORD_NO_PNC("CprNewDeliusRecordUnableToMatchNoPnc"),
+  NEW_DELIUS_RECORD_PNC_MATCHED("CprNewNDeliusRecordCreatedPNCMatched"),
+  NEW_DELIUS_RECORD_NEW_PNC("CprNewNDeliusRecordCreatedNewPNC"),
+  DELIUS_OFFENDER_READ_FAILURE("DeliusOffenderReadFailure"),
```

Comment:
I think this should be named `CprNDeliusOffenderReadFailure` to match the other events